packages/office-ui-fabric-react/src/components/Callout/examples/Callout.FocusTrap.Example.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
-import { FocusTrapCallout } from 'office-ui-fabric-react/lib/Callout';
-import { CommandBar, ICommandBarItemProps } from 'office-ui-fabric-react/lib/CommandBar';
-import { getId } from 'office-ui-fabric-react/lib/Utilities';
+import { DefaultButton } from 'office-ui-fabric-react';
+import { FocusTrapCallout } from 'office-ui-fabric-react';
+import { CommandBar, ICommandBarItemProps } from 'office-ui-fabric-react';
+import { getId } from 'office-ui-fabric-react';
```

Comment:
nit: these imports can be merged.

Reply:
yeah...they were originally deeper paths, and i changed them to root import at one point

Reply:
If you go back and change this at all, it would probably make sense to just merge *all* the OUFR imports (including the ones that are still deeper paths) to be from the root.